test/libyul/YulOptimizerTest.h

```diff
@@ -56,22 +56,15 @@ class YulOptimizerTest: public solidity::frontend::test::EVMVersionRestrictedTes
 	explicit YulOptimizerTest(std::string const& _filename);
 
 	TestResult run(std::ostream& _stream, std::string const& _linePrefix = "", bool const _formatted = false) override;
-
 private:
 	std::pair<std::shared_ptr<Object>, std::shared_ptr<AsmAnalysisInfo>> parse(
 		std::ostream& _stream, std::string const& _linePrefix, bool const _formatted, std::string const& _source
 	);
-	void disambiguate();
-	void updateContext();
-
 	static void printErrors(std::ostream& _stream, langutil::ErrorList const& _errors);
 
 	std::string m_optimizerStep;
 
 	Dialect const* m_dialect = nullptr;
-	std::set<YulString> m_reservedIdentifiers;
-	std::unique_ptr<NameDispenser> m_nameDispenser;
-	std::unique_ptr<OptimiserStepContext> m_context;
```

Comment:
Similarly for this.

Reply:
Actually can't lines 31-43 be removed as well?

Reply:
Cleaned up redundant includes and forward declarations.